tests/integrationtest/r/planner/core/indexmerge_path.result

```diff
@@ -499,11 +499,18 @@ TableReader_7	8000.00	root		data:Selection_6
 drop table if exists t;
 create table t(a int, j json, index kj((cast(j as signed array))));
 explain format = 'brief' select /*+ use_index(t, kj) */ * from t;
-Error 1815 (HY000): Internal : Can't find a proper physical plan for this query
+id	estRows	task	access object	operator info
+TableReader	10000.00	root		data:TableFullScan
+└─TableFullScan	10000.00	cop[tikv]	table:t	keep order:false, stats:pseudo
```

Comment:
This is a behavior change. Now it's more compatible with MySQL.

Reply:
make sense